lib/bellewaerde/index.js

```diff
@@ -0,0 +1,146 @@
+// include core Park class
+var Park = require("../park");
+
+var Moment = require("moment-timezone");
+var https = require("https");
+
+var rawRideData = require(__dirname + "/BellewaerdeData.js");
```

Comment:
```suggestion
var rawRideData = require("./BellewaerdeData.js");
```

Make this relative, otherwise it won't back-compile with Babel for older NodeJS versions (see Travis setup).